src/components/core/Alert.vue

```diff
@@ -16,23 +16,27 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-  <div v-if="alert">
-    <v-alert
-      :value="true"
-      :type="alert.color"
-      :icon="alert.icon"
-      :class="getColor(alert.color)"
-      dismissible
-      tile
-      light
-      colored-border
-    >
-      <template v-slot:close="props">
-        <v-icon @click="closeAlert(props.toggle)">{{ svgPaths.close }}</v-icon>
```

Comment:
`closeAlert` is no longer called so this method can be deleted now I think?

`closeAlert` did clear the store, wonder if this needs replacing somewhere. I'll have a look into this now.

Reply:
Did you come to any conclusions about this? I imagine I probably want to wire it back in?